detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -114,18 +129,26 @@ public Container GetContainer()
                                 Console.WriteLine($"Duplicate project name '{projectId}' found. Using GUID instead.");
                                 projectId = project.GUID;
                             }
+
+                            Boolean directoryPackagesExists = false;
                             Boolean projectFileExists = false;
                             try
                             {
                                 projectFileExists = File.Exists(projectPath);
+                                if (!projectFileExists)
```

Comment:
Add one more check for the Directory.Packages.props inside a folder and not skip when it exists.